cornice/validators/_marshmallow.py

```diff
@@ -163,3 +171,7 @@ def validator(request, schema=None, deserializer=None, **kwargs):
                 request.errors.add(location, location, details)
     else:
         request.validated.update(deserialized)
+
+
+def _instantiate_schema(schema):
```

Comment:
Here I would raise an exception if `schema` is not a class. Just in case developers switch code from colander to marshmallow for example

Reply:
Done